packages/core/stories/layout/deck-layout.stories.tsx

```diff
@@ -18,7 +18,7 @@ export default {
 const deckCards = (slides: number) =>
   Array.from({ length: slides }, (_, index) => (
     <Card key={index}>
-      <h2>{`Deck Item ${index + 1}`}</h2>
+      <h2 id="deck_item_title">{`Deck Item ${index + 1}`}</h2>
```

Comment:
This is producing duplicate Ids

Reply:
I've amended the Id although I've just realised we can't pass unique values to each `DeckItem` via `deckItemProps` e.g.`aria-labelledby` so if this is an ADA requirement we possibly need to extract `DeckItem` and allow it to be used separately

Reply:
Right now it reads all DeckItem contents. Card doesn't get focus so `aria-labelledby` isn't doing anything here

Reply:
Makes sense, I've removed this